doc/main/intro/testing_approach.rst

```diff
@@ -3,7 +3,7 @@
 Testing Approach 
 ================
 
-You can face four main hazard types in the libraries for parallelism:
+There are four main types of errors you can encounter in the development of libraries for parallelism:
```

Comment:
```suggestion
There are four main types of errors/hazard you can encounter in the development of libraries for parallelism:
```
Because for example 'Race conditions' is not a error it is typically hazard.